src/three-components/WebGLUtils.ts

```diff
@@ -75,3 +78,31 @@ export const applyExtensionCompatibility = (gl: WebGLRenderingContext) => {
     return extension;
   };
 };
+
+/**
+ * Debug method to save an offscreen render target to an image; filename should
+ * have a .png extension to ensure lossless transmission.
+ */
+export const saveTarget = (target: WebGLRenderTarget, filename: string) => {
```

Comment:
Can we move this to a separate module?

If we do that, it would be possible to trivially omit it from the deployed bundle in the typical case. Then, when we want it we can just import the module that has it during dev.

Reply:
K; do you have a filename in mind? And is there anything else similar that should go in the same one?

Reply:
Let's throw them in `utilities/debug.ts`. The shadow helper could also go there.